package.json

```diff
@@ -45,18 +49,28 @@
 		"node-fetch"
 	],
 	"devDependencies": {
+		"@sindresorhus/tsconfig": "^1.0.1",
+		"@types/body-parser": "^1.19.0",
+		"@types/busboy": "^0.2.3",
+		"@types/express": "^4.17.11",
+		"@types/node-fetch": "^2.5.8",
+		"@types/pify": "^5.0.0",
 		"abort-controller": "^3.0.0",
 		"ava": "^3.15.0",
-		"body": "^5.1.0",
+		"body-parser": "^1.19.0",
 		"busboy": "^0.3.1",
-		"create-test-server": "2.1.1",
-		"delay": "^4.1.0",
-		"form-data": "^3.0.0",
-		"node-fetch": "^2.5.0",
-		"nyc": "^15.0.0",
-		"playwright-chromium": "^1.7.1",
-		"tsd": "^0.13.1",
-		"xo": "^0.25.3"
+		"del-cli": "^3.0.1",
+		"delay": "^5.0.0",
+		"express": "^4.17.1",
```

Comment:
From what I can tell, the reason you're using Express directly is so that you can add types to the server. But why not just add types to the `create-test-server` package? Then the whole community can more easily benefit from that work and we don't have to maintain it in Ky.